src/Apps/Conversation/Components/ConversationList.tsx

```diff
@@ -27,12 +31,12 @@ const SpinnerContainer = styled.div`
 `
 
 const ConstrainedHeightContainer = styled(Flex)`
-  height: calc(100vh - 103px);
+  height: calc(100vh - ${DESKTOP_NAV_BAR_HEIGHT}px);
```

Comment:
There was a pixel added here which was causing the initial issue.

Reply:
No waaaay!!!! 🤯